include/pru/am572x.h

```diff
@@ -0,0 +1,769 @@
+		struct __PRU_INTC_HMR_S HMR0_bit;
+	};
+	union {
+		volatile unsigned int HMR1;
+		struct __PRU_INTC_HMR_S HMR1_bit;
+	};
+	union {
+		volatile unsigned int HMR2;
+		struct __PRU_INTC_HMR_S HMR2_bit;
+	};
+	unsigned int __gap11[61];
+	union {
+		volatile unsigned int HIPIR0;
+		struct __PRU_INTC_HIPIR_S HIPIR0_bit;
+	};
+	union {
+		volatile unsigned int HIPIR1;
+		struct __PRU_INTC_HIPIR_S HIPIR1_bit;
+	};
+	union {
+		volatile unsigned int HIPIR2;
+		struct __PRU_INTC_HIPIR_S HIPIR2_bit;
+	};
+	union {
+		volatile unsigned int HIPIR3;
+		struct __PRU_INTC_HIPIR_S HIPIR3_bit;
+	};
+	union {
+		volatile unsigned int HIPIR4;
+		struct __PRU_INTC_HIPIR_S HIPIR4_bit;
+	};
+	union {
+		volatile unsigned int HIPIR5;
+		struct __PRU_INTC_HIPIR_S HIPIR5_bit;
+	};
+	union {
+		volatile unsigned int HIPIR6;
+		struct __PRU_INTC_HIPIR_S HIPIR6_bit;
+	};
+	union {
+		volatile unsigned int HIPIR7;
+		struct __PRU_INTC_HIPIR_S HIPIR7_bit;
+	};
+	union {
+		volatile unsigned int HIPIR8;
+		struct __PRU_INTC_HIPIR_S HIPIR8_bit;
+	};
+	union {
+		volatile unsigned int HIPIR9;
+		struct __PRU_INTC_HIPIR_S HIPIR9_bit;
+	};
+	unsigned int __gap12[246];
+	volatile unsigned int SIPR0;
+	volatile unsigned int SIPR1;
+	unsigned int __gap13[30];
+	volatile unsigned int SITR0;
+	volatile unsigned int SITR1;
+	unsigned int __gap14[222];
+	union {
+		volatile unsigned int HINLR0;
+		struct __PRU_INTC_HINLR_S HINLR0_bit;
+	};
+	union {
+		volatile unsigned int HINLR1;
+		struct __PRU_INTC_HINLR_S HINLR1_bit;
+	};
+	union {
+		volatile unsigned int HINLR2;
+		struct __PRU_INTC_HINLR_S HINLR2_bit;
+	};
+	union {
+		volatile unsigned int HINLR3;
+		struct __PRU_INTC_HINLR_S HINLR3_bit;
+	};
+	union {
+		volatile unsigned int HINLR4;
+		struct __PRU_INTC_HINLR_S HINLR4_bit;
+	};
+	union {
+		volatile unsigned int HINLR5;
+		struct __PRU_INTC_HINLR_S HINLR5_bit;
+	};
+	union {
+		volatile unsigned int HINLR6;
+		struct __PRU_INTC_HINLR_S HINLR6_bit;
+	};
+	union {
+		volatile unsigned int HINLR7;
+		struct __PRU_INTC_HINLR_S HINLR7_bit;
+	};
+	union {
+		volatile unsigned int HINLR8;
+		struct __PRU_INTC_HINLR_S HINLR8_bit;
+	};
+	union {
+		volatile unsigned int HINLR9;
+		struct __PRU_INTC_HINLR_S HINLR9_bit;
+	};
+	unsigned int __gap15[246];
+	union {
+		volatile unsigned int HIER;
+		struct __PRU_INTC_HIER_S HIER_bit;
+	};
+};
+#define PRU_INTC	(*((struct __PRU_INTC *)__INTC_BASE))
+
+/* ====================== PRU UART module ====================== */
+/* TODO */
+
+/* ====================== PRU eCAP module ====================== */
+/* TODO */
+
+/* ====================== PRU CFG module ====================== */
+#define PRU_CFG_REVID_OFFS		0x00
+#define PRU_CFG_SYSCFG_OFFS		0x04
+#define PRU_CFG_GPCFG0_OFFS		0x08
+#define PRU_CFG_GPCFG1_OFFS		0x0c
+#define PRU_CFG_CGR_OFFS		0x10
+#define PRU_CFG_ISRP_OFFS		0x14
+#define PRU_CFG_ISP_OFFS		0x18
+#define PRU_CFG_IESP_OFFS		0x1c
+#define PRU_CFG_IECP_OFFS		0x20
+// #define PRU_CFG_SCRP_OFFS		0x24
```

Comment:
Please do not add dead code. Just remove it if it is no longer present.

FYI I also don't see SCRP in latest AM3 TRM.

Reply:
Done in 9b9e13b68f93c694947aacbfe69aa683abe0f3f8